homeassistant/components/rituals_perfume_genie/config_flow.py

```diff
@@ -27,7 +30,7 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_user(self, user_input=None) -> FlowResult:
+    async def async_step_user(self, user_input: ConfigType | None = None) -> FlowResult:
```

Comment:
This is not YAML config that is passed here, thus not a `ConfigType`

```suggestion
    async def async_step_user(self, user_input: dict[str, Any] | None = None) -> FlowResult:
```